website/blog/2021-06-24-introduce-superbench.md

```diff
@@ -8,4 +8,113 @@ author_url: https://github.com/TobeyQin
+Empowered by SuperBench CLI, user can deploy and run their benchmarks with only one command, which greatly shorten the learning time of using tools, 
+help user easily evaluate the performance of AI workload.
+
+Below is a simple example to show how to deploy and run benchmarks locally. For more information, 
+please view [CLI Document](https://github.com/microsoft/superbenchmark/blob/main/docs/cli.md)
+
+1. Deploy
+
+    ```bash
+    sb deploy -f local.ini
+    ```
+2. Run Benchmark
+
+    ```bash
+    sb run -f local.ini -c config.yaml
+    ```
+
+Among them, `local.ini` is the configuration file to manage nodes. If you want to run benchmarks locally, the node is `localhost`.
+
+```ini title="local.ini"
+[all]
+localhost ansible_connection=local
+```
+`config.yaml` is a config file to configure the details of benchmarkings. You can customize your benchmarks by modified this file.
+
+For more information, please view [confiuration](https://microsoft.github.io/superbenchmark/docs/getting-started/configuration)
+
+### Modular and Extensible Framework
+
+1. Executor Framework
+
+In order to facilitate the benchmarking and validation of large-scale clusters, we designed and implemented a modular and extensible framework. 
+SuperBench framework includes a runner as control node, as well as multiple executors as worker nodes. 
+A runner received commands from CLI and distribute to all nodes (worker nodes) in the cluster and collect results. 
+Worker nodes will execute the specified benchmark tasks.
+
+![SuperBench Executor Workflow](../../docs/assets/executor_workflow.png)
+
+2. Benchmark Framework
+
+SuperBench supports micro-benchmark for primitive computation and communication benchmarking,
+and model-benchmark to measure domain-aware end-to-end deep learning workloads.
+SuperBench implemented an abstract BenchmarkBase to provide common fuction, while other benchmarks are plugged-in as components.
+It also provides a unified entrance and result format for all benchmarks.
+Developers can easily add new benchmarks.
+
+![SuperBench Benchmark Package](../../docs/assets/benchmark_package.png)
+
+### Conprehensive and Strandardized Benchmarks
+
+SuperBench supports a set of benchmarks listed as below.
+
+* Micro-Benchmarks
+  * Computation benchmarks
+    * GEMM Flops
+    * Kernel Launch Time
+    * Operator Performance
+    * Memory
+  * Communication benchmarks
+    * Device P2P
+    * RDMA
+    * NCCL
+  * Computation-Communication Benchmarks
+  * Storage Benchmarks
+
+* Model-Benchmarks
+  * CNN models
+  * LSTM models
+  * BERT models
+  * GPT-2 models
+
+For the details of each benchmark, please view [micro-benchmarks](https://github.com/microsoft/superbenchmark/blob/main/docs/benchmarks/micro-benchmarks.md) 
+and [model-benchmarks](https://github.com/microsoft/superbenchmark/blob/main/docs/benchmarks/model-benchmarks.md).
```

Comment:
same, use website link?

Reply:
Currently we don't have the website link yet. Maybe I can add them to the website, then we can change to the website link.

Reply:
If you have updated benchmarks md files, you can uncomment the following lines to add them on website:
https://github.com/microsoft/superbenchmark/blob/f22bb3f219e6ca6ea30ac6ba9923634d2510537b/website/sidebars.js#L18-L25

Reply:
Changed